src/OpenPhySyn/src/StandardTransforms/BufferFanoutTransform/CMakeLists.txt

```diff
@@ -32,27 +30,19 @@ if(NOT PSN_LIB_PATH STREQUAL 0)
 link_directories(${PSN_LIB_PATH})
 endif()
 
-if(${OPENPHYSYN_TRANSFORM_AUTO_LINK_TRANSFORMS})
 add_library(${TRANSFORM_NAME}  src/BufferFanoutTransform.cpp)
 target_compile_definitions(${TRANSFORM_NAME} INTERFACE OPENPHYSYN_TRANSFORM_BUILD)
-else()
-add_library(${TRANSFORM_NAME} SHARED src/BufferFanoutTransform.cpp)
-endif()
 
 target_include_directories(${TRANSFORM_NAME} PUBLIC
   src/
   ${PSN_HOME}/include/
-  ${PSN_HOME}/external/spdlog/include
   ${OPENDB_HOME}/include
-  ${OPENDB_HOME}/src
-  ${OPENDB_HOME}/src/db
-  ${OPENSTA_HOME}/..
+  ${OPENSTA_HOME}
```

Comment:
Its better to use target_link_library(${TRANSFORM_NAME} OpenSTA) rather than manually manipulate the include paths.  Here and elsewhere.

Reply:
Done